components/Link.js

```diff
@@ -1,14 +1,31 @@
 import { html } from 'https://unpkg.com/rplus-production@1.0.0';
 import shouldPreventDefault from '../utils/shouldPreventDefault.js';
 
-export default ({ href, onClick, innerText }) => {
+const pushState = url => history.pushState(null, null, url);
+
+/** Link component props:
+ * href = anchor href
+ * innerText = text to display between anchor tags
+ * customOnClick = a function (or boolean) that:
+ * - disables the pushState function, and calls `customOnClick` instead, if it's a function.
+ * - Allows you to also pass in `true` to simply disable `pushState` with no alternative function
+ * noPreventDefault = falsy by default, set to `true` to disable our `preventDefault()`
```

Comment:
So `noPreventDefault` to true essentially makes this an `a` tag?

Reply:
`noPreventDefault` in combination with `customOnClick` (i.e. both of these being `true`) would mean nothing runs `onClick` and make it act like a typical `a`, yes!